lib/hydrothermalVenture.ex

```diff
@@ -0,0 +1,42 @@
+defmodule AdventCode2 do
+  def createRange([{a, b}, {a, c}]) do
+    Enum.to_list(b..c)
+    |> Enum.map(fn x -> {a, x} end)
+  end
+
+  def createRange([{a, b}, {c, b}]) do
+    Enum.to_list(a..c)
+    |> Enum.map(fn x -> {x, b} end)
```

Comment:
```suggestion
    Enum.map(a..c, & {&1, b})
```

[Captures](https://hexdocs.pm/elixir/Kernel.SpecialForms.html#&/1) я лично люблю, вы тоже к ним привыкнете.